pkg/machines/components/vcpuModal.css

```diff
@@ -0,0 +1,17 @@
+.vcpu-modal-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  grid-row-gap: 1em;
+}
+
+.vcpu-modal-grid input[type=number] {
+    max-width: 7rem;
```

Comment:
Used `rem`s instead of px, so make it obvious it's keyed on default font size. (Perhaps these should be `em`s? At least `rem`s are more consistent. With PF in Cockpit, `1rem` == `10px`.)

Reply:
Also used max-width to limit the width, instead of width.